### PR TITLE
Add a converter String->Expession

### PR DESCRIPTION
**Pull request by ericbottard**
Modules that want to use this just have to import the configuration

### CODE REVIEW

**Comment by markfisher** [comment]
Can this go in the 'config' package rather than creating a new one?

**Comment by ericbottard** [comment]
Sure thing, did not want to pollute the thing related to pure binding.

**Comment by markfisher** [comment]
Ultimately, I think this should go into the default converters used by the BeanFactory. Maybe issue a PR for that?

**Comment by ericbottard** [comment]
Will do. I don't see how it could hurt indeed

File: spring-cloud-stream/src/main/java/org/springframework/cloud/stream/config/SpelExpressionConverterConfiguration.java [hunk 1]
```diff
@@ -0,0 +1,60 @@
+/*
+ * Copyright 2015 the original author or authors.
```
Comment (markfisher) [inline spring-cloud-stream/src/main/java/org/springframework/cloud/stream/config/SpelExpressionConverterConfiguration.java]: need an empty line after the "Copyright..."

**Comment by markfisher** [comment]
rebased, squashed, and merged as https://github.com/spring-cloud/spring-cloud-stream/commit/2ab2d5c1a904a99929ec5f3d4f44cd6b369677f1

**Pull request closed without merging**